src/parameterizations/stochastic/MOM_stochastics.F90

```diff
@@ -18,28 +22,56 @@ module MOM_stochastics
 use mpp_domains_mod,         only : mpp_define_domains, mpp_get_compute_domain, mpp_get_data_domain
 use MOM_domains,             only : root_PE, num_PEs
 use MOM_coms,                only : Get_PElist
+use MOM_EOS,                 only : calculate_density, EOS_domain
 use stochastic_physics,      only : init_stochastic_physics_ocn, run_stochastic_physics_ocn
 
 #include <MOM_memory.h>
 
 implicit none ; private
 
-public stochastics_init, update_stochastics
+public stochastics_init, update_stochastics, apply_skeb
 
 !> This control structure holds parameters for the MOM_stochastics module
 type, public:: stochastic_CS
   logical :: do_sppt         !< If true, stochastically perturb the diabatic
+  logical :: do_skeb         !< If true, stochastically perturb the diabatic
```

Comment:
Neither the (identical) descriptions of what `do_sppt` nor `do_skeb` mean are very informative to me.  I think that they should both be revised for clarity and to differentiate between them.